### DISCUSSION TITLE
why I can't update pnpm version to 7.5.0 with corepack?

### DISCUSSION THREAD

**Discussion by shervinchen** [pnpm/pnpm#4994] [Q&A]
I have followed the official documentation and run:
`corepack enable`
`corepack prepare pnpm@7.5.0 --activate`
but the terminal just console:
`Preparing pnpm@7.5.0 for immediate activation...`
and nothing happened...
Is it because the latest version of pnpm hasn't been released to corepack yet?

**Comment by shervinchen** [comment]
It seems I can't update pnpm version alone, because it is in corepack?
I try to update corepack version to 0.11.2(latest version) and the pnpm version is update to 7.2.1.
So when the pnpm version is updated, corepack will not update the version immediately, which means you can only go update the corepack version periodically?

**Reply by zkochan** [reply]
I don't really know. You should probably ask in the corepack repo: https://github.com/nodejs/corepack

**Comment by rohanrajpal** [comment]
I was facing the issue but quickly realised that I had pinned the pnpm version in my package json. So updating `pnpm` should be as simple as just changing the following line in your `package.json`

```
  "packageManager": "pnpm@7.6.0",
```
